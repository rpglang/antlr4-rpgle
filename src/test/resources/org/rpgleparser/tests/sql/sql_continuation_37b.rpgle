      // =======================================================================
      // - SQL Compiler Options -
      // =======================================================================
     C/exec sql
     C+ set option closqlcsr = *endactgrp,
     C+            commit    = *none,
     C+            alwblk    = *allread,
     C+            dlyprp    = *yes,
     C+            alwcpydta = *optimize,
     C+            rdbcnnmth = *duw,
     C+            naming    = *sys,
     C+            datfmt    = *iso,
     C+            timfmt    = *hms
     C/end-exec