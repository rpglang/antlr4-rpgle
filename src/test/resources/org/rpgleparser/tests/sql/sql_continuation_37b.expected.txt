/*===TOKENS===*/
HIDDEN:COMMENTS          < //>
HIDDEN:COMMENTS_TEXT     <=======================================================================>
HIDDEN:COMMENTS          < //>
HIDDEN:COMMENTS_TEXT     <- SQL Compiler Options ->
HIDDEN:COMMENTS          < //>
HIDDEN:COMMENTS_TEXT     <=======================================================================>
CS_ExecSQL               <C/exec sql>
CSQL_TEXT                < set option closqlcsr = *endactgrp,                                                         >
CSQL_TEXT                <            commit    = *none,                                                              >
CSQL_TEXT                <            alwblk    = *allread,                                                           >
CSQL_TEXT                <            dlyprp    = *yes,                                                               >
CSQL_TEXT                <            alwcpydta = *optimize,                                                          >
CSQL_TEXT                <            rdbcnnmth = *duw,                                                               >
CSQL_TEXT                <            naming    = *sys,                                                               >
CSQL_TEXT                <            datfmt    = *iso,                                                               >
CSQL_TEXT                <            timfmt    = *hms                                                                >
CSQL_END                 <C/end-exec>
/*===TREE===*/
(r
  (statement
    (cspec_fixed_sql
      C/exec sql
      ' set option closqlcsr = *endactgrp,                                                         '
      '            commit    = *none,                                                              '
      '            alwblk    = *allread,                                                           '
      '            dlyprp    = *yes,                                                               '
      '            alwcpydta = *optimize,                                                          '
      '            rdbcnnmth = *duw,                                                               '
      '            naming    = *sys,                                                               '
      '            datfmt    = *iso,                                                               '
      '            timfmt    = *hms                                                                '
      C/end-exec
    )
  )
)
/*======*/